sdk/src/main/scala/com/horizen/utils/WithdrawalEpochUtils.scala

```diff
@@ -1,37 +1,61 @@
 package com.horizen.utils
```

Comment:
Same here, just an auto-reformat to reduce line length and increase readability.